src/method.jl

```diff
@@ -0,0 +1,115 @@
+"""
+    signature(meth::Method) -> Dict{Symbol,Any}
+
+Finds the expression for a methods signature as broken up into its various components
+including:
+
+- `:head`: Expression head of the function definition: always `:function`
+- `:name`: Name of the function (not present for anonymous functions)
+- `:params`: Parametric types defined on constructors
+- `:args`: Positional arguments of the function
+- `:rtype`: Return type of the function
```

Comment:
Can we retrieve this?

Reply:
I would like to, I need to work out how.

Reply:
Oh that does not work. need the `@code_typed` CodeInfo

Reply:
I have added to the "Will not be returned" list